TODO

```diff
@@ -181,8 +181,17 @@ Features:
 * systemd-sysupdate: make transport pluggable, so people can plug casync or
   similar behind it, instead of http.
 
-* systemd-tmpfiles: add concept for conditionalizing lines on factory reset
-  boot, or on first boot.
+* systemd-tmpfiles: add concept for conditionalizing lines on first boot.
+
+* systemd-tmpfiles: check the kernel cmdline and EFI variable for factory reset
+  like systemd-repart does, and use it even when --factory-reset=no
```

Comment:
hmm, no, if --factory-reset=no is set, we should *not* check the efi var and kernel cmdline.

if --factory-reset= is explicitly specified it should always take precedence. Only if it is not specified we should check efi var/kernel cmdline